controllers/cluster.js

```diff
@@ -727,5 +727,36 @@ router.get('/:node_id/configuration/validation', cache(), function(req, res) {
     execute.exec(python_bin, [wazuh_control], data_request, function (data) { res_h.send(req, res, data); });
 })
 
+/**
+ * @api {get} /cluster/:node_id/config/:component/:configuration Get active configuration in node node_id
+ * @apiName GetClusterNodeActiveConfiguration
+ * @apiGroup Configuration
+ *
+ * @apiParam {String="agent","agentless","analysis","auth","com","csyslog","integrator","logcollector","mail","monitor","request","syscheck","wmodules"} [component] Indicates the wazuh component to check.
```

Comment:
SCA module is missed in this list. We should include it and check if we can get that configuration on demand via socket.

Reply:
Hello @crd1985,

SCA is already included inside `wmodules`  component and `wmodules` configuration:
```javascript
# curl -u foo:bar "localhost:55000/agents/000/config/wmodules/wmodules?pretty"
{
   "error": 0,
   "data": {
      "wmodules": [
         {
            "sca": {
               "enabled": "yes",
               "scan_on_start": "yes",
               "skip_nfs": "yes",
               "interval": 43200,
               "policies": [
                  "cis_debian_linux_rcl.yml",
                  "system_audit_rcl.yml",
                  "system_audit_ssh.yml"
               ]
            }
         }
      ]
   }
}
```